kolibri/core/logger/serializers.py

```diff
@@ -59,13 +60,13 @@ def update(self, instance, validated_data):
             instance.completion_timestamp = now()
         instance = super(ExamLogSerializer, self).update(instance, validated_data)
         # to check if a notification must be created:
-        wrap_to_save_queue(parse_examlog, instance)
+        wrap_to_save_queue(parse_examlog, instance, local_now())
         return instance
 
     def create(self, validated_data):
         instance = super(ExamLogSerializer, self).create(validated_data)
         # to check if a notification must be created:
-        wrap_to_save_queue(create_examlog, instance)
+        wrap_to_save_queue(create_examlog, instance, local_now())
```

Comment:
So, the problem came from the fact the queue saving the notification can have delays, thus timestamp=local_now is not the logger timestamp, but the 'saving notification' timestamp. If that's right this PR makes totally sense.

Reply:
Yeah, I think that was definitely causing some issues with timestamping.